language/unit_tests/test_document.py

```diff
@@ -308,13 +308,164 @@ def test_analyze_sentiment(self):
         document = self._make_one(client, content)
 
         sentiment = document.analyze_sentiment()
-        self._verify_sentiment(sentiment, polarity, magnitude)
+        self._verify_sentiment(sentiment, score, magnitude)
 
         # Verify the request.
         expected = self._expected_data(content)
         client._connection.api_request.assert_called_once_with(
             path='analyzeSentiment', method='POST', data=expected)
 
+    def _verify_token(self, token, text_content, part_of_speech, lemma):
+        from google.cloud.language.syntax import Token
+        from google.cloud.language.syntax import PartOfSpeech
+
+        self.assertIsInstance(token, Token)
+        self.assertEqual(token.text_content, text_content)
+        self.assertEqual(token.part_of_speech, part_of_speech)
+        self.assertEqual(token.lemma, lemma)
+
+    def test_analyze_syntax(self):
+        from google.cloud.language.document import Encoding
+        from google.cloud.language.syntax import Token
+        from google.cloud.language.syntax import PartOfSpeech
+
+        name1 = 'R-O-C-K'
+        name2 = 'USA'
+        content = name1 + ' in the ' + name2
+        response = {
+            'sentences': [
+                {
+                    'text': {
+                        'content': 'R-O-C-K in the USA',
+                        'beginOffset': -1,
+                    },
+                    'sentiment': None,
+                }
+            ],
+            'tokens': [
+                {
+                    'text': {
+                        'content': 'R-O-C-K',
+                        'beginOffset': -1,
+                    },
+                    'partOfSpeech': {
+                        'tag': 'NOUN',
+                        'aspect': 'ASPECT_UNKNOWN',
+                        'case': 'CASE_UNKNOWN',
+                        'form': 'FORM_UNKNOWN',
+                        'gender': 'GENDER_UNKNOWN',
+                        'mood': 'MOOD_UNKNOWN',
+                        'number': 'SINGULAR',
+                        'person': 'PERSON_UNKNOWN',
+                        'proper': 'PROPER',
+                        'reciprocity': 'RECIPROCITY_UNKNOWN',
+                        'tense': 'TENSE_UNKNOWN',
+                        'voice': 'VOICE_UNKNOWN',
```

Comment:
Is **all** of this content necessary in the unit test? It's almost certainly going to make `pylint` cry to its mother